mobilizon_bots/storage/query.py

```diff
@@ -48,6 +48,15 @@ async def get_unpublished_events() -> Iterable[MobilizonEvent]:
     return await events_with_status([PublicationStatus.WAITING])
 
 
+async def get_mobilizon_event_publications(
+    event: MobilizonEvent,
+) -> Iterable[Publication]:
+    models = await prefetch_event_relations(
+        Event.filter(mobilizon_id=event.mobilizon_id)
+    )
```

Comment:
Yes it would, bit first() doesn't return a `QuerySet`, it returns a subclass of a `Model`, so we would need to implement a version of prefetch_event_relations that works with models instead of queries